spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -303,7 +301,7 @@ class SpdxExpressionTest : WordSpec() {
 
             "not merge license-exception pairs with single matching licenses" {
                 "GPL-2.0-or-later WITH Classpath-exception-2.0 AND GPL-2.0-or-later"
-                    .decompose() shouldContainExactlyInAnyOrder listOf(
+                    .decompose() should containExactlyInAnyOrder(
```

Comment:
Wrong indentation below. I acknowledge it was broken before, but I believe it makes sense to fix it now.

Reply:
Same as above.

Reply:
Done.